src/app/modules/time-entries/pages/time-entries.component.html

```diff
@@ -54,8 +54,14 @@
             <th class="col"></th>
           </tr>
         </thead>
+        <tr *ngIf="dataSource.isLoading">
```

Comment:
Could you make a function to check this, because you have two if condition that check if the element must show the message

Reply:
You're right, that's a better approach. I am going to make that function, thank you.